be/src/exec/pipeline/operator.h

```diff
@@ -252,8 +264,11 @@ class Operator {
 
     RuntimeBloomFilterEvalContext _bloom_filter_eval_context;
 
-    // if _need_spill is true. reserved data in this operator need spill
-    bool _marked_need_spill = false;
+    inline static size_t MAX_PERFORMANCE_LEVEL = 3;
+    // performance level. Determine the execution mode and whether memory can be freed early
+    // A higher performance level will allow the operator to execute with less memory, which will reduce performance
+    size_t _performance_level{};
```

Comment:
1. need to give a clear behavior, what kind of things will be done under the value of each performance_leve;
2. should this concept belong to the query level? I'm trying to release the buffer data when memory resources are not sufficient... these are all strategies to reduce memory and need such a similar concept

Reply:
This variable is designed for just such a scenario. Sometimes we just need to keep the memory in the operator from growing, but other times we need to release all the memory in the operator as soon as possible.